### PR TITLE
Update pipeline apps for logging RFC

### PR DESCRIPTION
**Pull request by warrd**
## Issues

https://github.com/wellcomecollection/platform/issues/4435
https://github.com/wellcomecollection/platform/issues/4460

### CODE REVIEW

File: infrastructure/modules/worker_with_sidecar/main.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  namespace_id       = var.namespace_id
+  security_group_ids = var.security_group_ids
+  use_fargate_spot   = var.use_fargate_spot
+
+  image           = var.app_image
+  port            = var.app_port
+  env_vars        = var.app_env_vars
+  secret_env_vars = var.app_secret_env_vars
+  app_cpu         = var.app_cpu
+  app_memory      = var.app_cpu
```
Comment (warrd) [inline infrastructure/modules/worker_with_sidecar/main.tf:21]: typo

File: infrastructure/modules/worker/variables.tf [hunk 2]
```diff
@@ -66,3 +76,14 @@ variable "security_group_ids" {
   type    = list(string)
   default = []
 }
+
+variable "port" {
+  // Leaving this as the default will not expose a port on the container
```
Comment (alexwlchan) [inline infrastructure/modules/worker/variables.tf:81]: 👍 useful comment

File: pipeline/terraform/modules/service_with_manager/main.tf [hunk 3]
```diff
@@ -1,62 +1,37 @@
...
+  app_env_vars        = var.app_env_vars
+  app_secret_env_vars = var.app_secret_env_vars
+  app_port            = var.app_container_port
+
+  sidecar_image           = var.manager_container_image
+  sidecar_name            = var.manager_container_name
+  sidecar_cpu             = var.manager_cpu
+  sidecar_memory          = var.manager_memory
+  sidecar_env_vars        = var.manager_env_vars
+  sidecar_secret_env_vars = var.secret_manager_env_vars
```
Comment (alexwlchan) [inline pipeline/terraform/modules/service_with_manager/main.tf:30]: nit: manager_secret_env_vars for consistency?

File: pipeline/terraform/modules/service_with_manager/main.tf [hunk 4]
```diff
@@ -1,62 +1,37 @@
...
-  sidecar_container_name           = var.manager_container_name
-  sidecar_cpu                      = var.manager_cpu
-  sidecar_memory                   = var.manager_memory
-  sidecar_env_vars                 = var.manager_env_vars
-  secret_sidecar_env_vars          = var.secret_manager_env_vars
-  sidecar_depends_on_app_condition = var.app_healthcheck_json == "" ? "" : "HEALTHY"
+  app_name            = var.app_container_name
+  app_image           = var.app_container_image
+  app_cpu             = var.app_cpu
+  app_memory          = var.host_memory
```
Comment (alexwlchan) [inline pipeline/terraform/modules/service_with_manager/main.tf:20]: host_memory?

**Pull request merged**